app/src/ui/lib/list/list.tsx

```diff
@@ -419,6 +419,10 @@ export class List extends React.Component<IListProps, IListState> {
       this.props.onRowKeyDown(rowIndex, event)
     }
 
+    if (event.defaultPrevented) {
```

Comment:
I think this change is fine, but since this touches a core component, I'd like to get the thoughts of someone else from @desktop/engineering before giving this a full ✅

Reply:
i'm not sure this `if` statement is really necessary. it looks like similar logic (on [L430](https://github.com/desktop/desktop/pull/7840/files#diff-a6fd50f3585afc2eeb1a60931fbf01a8R430)) should already take care of this. @HashimotoYT is there more context here that i'm missing?